bin/bst-utter.ts

```diff
@@ -60,7 +61,11 @@ program
             url = "http://localhost:" + proxyProcess.port;
         }
 
-        const speaker = new BSTVirtualAlexa(url, interactionModel, intentSchemaPath, samplesPath, applicationID, locale);
+        const savedSession = Global.config().loadSession();
```

Comment:
this can return null if the file is not present, (as it would be on a first run of the tool), and then savedSession.locale would throw an exception.

Add a validation an tests for that scenario